src/index.js

```diff
@@ -1,30 +1,36 @@
-import VanillaCaret from "vanilla-caret-js";
-import Observer from "./observer";
+import VanillaCaret from 'vanilla-caret-js';
+import { diff, unpatch, patch } from 'jsondiffpatch';
+// eslint-disable-next-line import/no-unresolved
+import * as jsonpatchFormatter from 'jsondiffpatch/formatters/jsonpatch';
+import Observer from './observer';
 
 /**
  * Undo/Redo feature for Editor.js.
  *
  * @typedef {Object} Undo
  * @description Feature's initialization class.
  * @property {Object} editor — Editor.js instance object.
- * @property {Number} maxLength - Max amount of changes recorded by the history stack.
+ * @property {Number} maxLength - Max amount of changes recorded by the history undoStack.
  * @property {Function} onUpdate - Callback called when the user performs an undo or redo action.
- * @property {Boolean} shouldSaveHistory - Defines if the plugin should save the change in the stack
+ * @property {Boolean} shouldSaveHistory - Defines if the plugin should save the change in the undoStack
  * @property {Object} initialItem - Initial data object.
+ * @property {Object} basicData - Saved data object.
```

Comment:
Actually, I preferred the `baseData` name, is there a reason why you decided to change it?

Reply:
Yes, we need to change that name, maybe the `basicData` confuses because it sounds like it was only a part of the expected data